core/backend/package.json

```diff
@@ -22,11 +22,10 @@
     "lint": "eslint -f visualstudio \"./src/**/*.ts\" 1>&2",
     "pretest": "cpx ./src/test/logging.config.json ./lib/test",
     "test": "betools test --offline=\"mock\" --grep=\"#integration|#WebGLPerformance\" --invert",
-    "runIntegrationTests": "betools test --testDir=\"./lib/test/integration\"",
-    "setupAzurite": "mkdir .\\azuriteStorage && azurite --silent --location ./azuriteStorage",
-    "cleanupAzurite": "rimraf ./azuriteStorage",
-    "runIntegrationTestsWithAzurite": "run-p -r setupAzurite runIntegrationTests",
-    "test:integration": "run-s -c pretest runIntegrationTestsWithAzurite cleanupAzurite",
+    "azurite:setup": "mkdir ./lib/test/azuriteStorage && azurite --silent --location ./lib/test/azuriteStorage",
```

Comment:
@skirby1996 another issue I just hit with Azurite is port conflicts. To be fair I was running it on two separate source trees at the same time :)